config/inertia.php

```diff
@@ -23,7 +23,7 @@
 
         'enabled' => true,
 
-        'url' => 'http://127.0.0.1:13714',
+        'url' => 'http://127.0.0.1:' . env('VITE_INERTIA_SSR_PORT', 13714),
```

Comment:
Suggest adding an entry in `.env.sample`.

Reply:
Wasn't sure whether to do this or not -- updated in latest.